src/modules/utils.js

```diff
@@ -0,0 +1,106 @@
+import TodoList from './todoList.js';
+
+const todo = new TodoList();
+const todoList = document.getElementById('dynamic-list');
+const form = document.getElementById('form');
+const allTasks = () => {
+  todoList.innerHTML = '';
+  todo.todos
+    .sort((a, b) => a.index - b.index)
+    .map((todos) => {
+      const todoCard = document.createElement('form');
+      todoCard.classList.add('one');
+      todoCard.setAttribute('id', todos.id);
+      const checkBox = document.createElement('input');
+      checkBox.setAttribute('type', 'checkbox');
+      if (todos.completed) {
+        checkBox.setAttribute('checked', 'checked');
+      }
+      checkBox.addEventListener('change', () => onchange(todos));
+      todoCard.appendChild(checkBox);
+      const description = document.createElement('p');
+      description.innerHTML = todos.description;
+      todoCard.appendChild(description);
+      const button = document.createElement('button');
+      button.onclick = () => editDescription(todos);
+      const icon = document.createElement('i');
+      icon.classList.add('fas', 'fa-ellipsis-v');
+      button.appendChild(icon);
+      todoCard.appendChild(button);
+      todoList.appendChild(todoCard);
```

Comment:
- [ ] Adding, removing, and editing a task works 💯 keep rocking 👍. However, let's add a line through the description once a task is checked as completed. There is a CSS property you can use to achieve this, check [here](https://www.w3schools.com/cssref/pr_text_text-decoration.asp) to learn about it. This is part of the requirement and it also improves user experience. See what I mean below:

Your Implementation | Required implementation
 :---:|:---:
 ![](https://i.imgur.com/ERxUKPj.png) | ![](https://i.imgur.com/daKT431.png)

- [ ] Let's also add instructions on how to start your app in the README so that users can find all the commands needed to start your app. Add commands like `npm install`, `npm start` under the `Install` section of the README 😉.

Reply:
Thank you @julie-ify i implemented the changes